docs/celo-codebase/protocol/stability/adding_stable_assets.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Add more stable assets
+---
+
+This document outlies the requirements and steps to add a new stable asset to the Celo platform. Assuming we want to add to the platform a new stable asset `cX` tracking the value of X (where X can be a fiat currency like ARS or MXN), using the [Mento exchange](doto.md).
+
+## Requirements
+
+1. Liquidity: the asset X has to be liquidly traded against CELO, in a CELO/X ticker. In absence of that, X has to be liquidly traded, including weekends, against well known assets that trade 24/7, like BTC or ETH, such that the price of X with respect to Celo can be inferred. In this second case, an implicit pair can be calculated for the oracle reports.
+2. Determine pre-mint addresses and amounts: It is possible to pre-mint a fixed amount at the time of launching a new stable asset, good candidates to receive the pre-mint are the community fund and other entities commited to distribute this initial allocation to grant recipients and liquidity providers.
+
+A good criteria to a successfully decide a pre-mint amount is to check by how much it would affect the reserve collateralization ratio, this is, the ratio of all stable assets, divided by all the reserve holdings. Reserve information, as well as the collateralization ration can be found on the [Reserve website](https://celoreserve.org/).
+## Procedure
+
+### Including contracts on the registry
+
+Currently, the addition of new assets is tied to the [Contract Release Cycle](../../../community/release-process/smart-contracts), as the contracts `ExchangeX` and `StableTokenX` need to be checked in [^1]. These new contracts inherit from Exchange and StableToken, that are the ones originally used for `cUSD`. As StableToken `cX` will be initialized by the contract release, key parameters like `spread` and `reserveFraction` should be included, although they can be later modified by setters in the following governance proposals. The only value that can't be changed is the pre-mint amount.
```

Comment:
@dckesler i modified the link here to be relative rather than absolute. the linked page doesn't have the sidebar, this is still TODO, correct?

Reply:
also, the ^1 link isn't working for me.

Reply:
Yeah this won't work probably until we get the community docs converted to docusaurus. See issue #8